src/js/contexts/AnnounceContext/AnnounceContext.js

```diff
@@ -16,8 +17,9 @@ export const AnnounceContext = React.createContext(
   (message, mode = 'polite', timeout = 500) => {
     // we only create a new container if we don't have one already
     // we create a separate node so that grommet does not set aria-hidden to it
+    const id = 'grommet-announcer';
```

Comment:
Seems inconsistent to take `id` as an argument but then hard code it here. Should we remove the argument and move line 20 up to line 4?